python/interpret_community/common/explanation_utils.py

```diff
@@ -360,19 +345,6 @@ def _unsort_1d(values, order):
     return np.array(values)[order_inverse]
 
 
-def _unsort_2d(values, order):
```

Comment:
_unsort_2d is used by explanation client :( we should make it public perhaps

Reply:
Thanks for pointing this. Since this function is not used in interpret-community, I have ported the code to azureml-interpret. This can be cleaned up now.

Reply:
sounds great!